src/app/app.component.html

```diff
@@ -1,21 +1,37 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
-  <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
-</div>
-<h2>Here are some links to help you start: </h2>
-<ul>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of Heroes</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://angular.io/cli">CLI Documentation</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
+<nb-layout>
 
-<router-outlet></router-outlet>
+  <nb-layout-header>
+    <button nbButton (click)="toggleTheme()">Toggle Theme</button>
+  </nb-layout-header>
+
+  <nb-layout-column>
+
+    <nb-card>
+      <nb-card-header>
+        <input nbInput fieldSize="giant" fullWidth placeholder="Add item" #input (keyup.enter)="onAdd(input.value); input.value = ''" />
+      </nb-card-header>
+
+      <nb-list>
+        <nb-list-item *ngFor="let item of items$ | async">
+          <app-todo-item [item]="item"
+                         (save)="onSave($event)"
+                         (toggleComplete)="onToggleCompleted($event)"
+                         (delete)="onDelete($event)">
+          </app-todo-item>
+        </nb-list-item>
+
+        <nb-list-item *ngIf="(items$ | async).length === 0" class="no-items">No items yet.</nb-list-item>
+      </nb-list>
+
+      <nb-card-footer>
+        <span> {{ (items$ | async).length }} items</span>
```

Comment:
The same here, it'll make templates much easier to scan
```suggestion
        <span> {{ numberOfItems$ }} items</span>
```

Reply:
does it simplify things or make it more obvious or helps to reuse it?